kura/org.eclipse.kura.ble.provider/META-INF/MANIFEST.MF

```diff
@@ -6,7 +6,7 @@ Bundle-Version: 1.0.100
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: org.apache.commons.io;version="[2.4,3.0)",
  org.eclipse.kura;version="[1.0,2.0)",
- org.eclipse.kura.bluetooth.le;version="[1.0,1.1)",
+ org.eclipse.kura.bluetooth.le;version="[1.1.0,1.2.0)",
  org.eclipse.kura.bluetooth.le.beacon;version="[1.0,1.1)",
  org.eclipse.kura.bluetooth.le.beacon.listener;version="1.0.0",
```

Comment:
as before

Reply:
Of course, Eclipse IDE manages the version in a different way... I'll edit the files manually.